.github/workflows/android.yml

```diff
@@ -11,11 +11,11 @@ jobs:
    runs-on: ubuntu-latest
    steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-java@v2
+    - uses: actions/setup-java@v3
       with:
         distribution: 'zulu'
         java-version: '11'
```

Comment:
you're building with java 11 (class file v55.0) when java 17 seems to be needed (class file v61.0)